Cargo.toml

```diff
@@ -159,7 +159,7 @@ windows = { version = "0.56", features = [
     "Win32_NetworkManagement_IpHelper",
 ] }
 
-boringtun = { git = "https://github.com/NordSecurity/boringtun.git", tag = "v1.2.2", features = ["device"] }
+boringtun = { git = "https://github.com/NordSecurity/boringtun.git", tag = "v1.2.3", features = ["device"] }
```

Comment:
Well, we have another PR which wants to bump boringtun to v1.2.3 ^^' : https://github.com/NordSecurity/libtelio/pull/673/files

Reply:
Asked Matas about it, he said he'll bump his version to 1.2.4, so this PR can be merged.